react/features/base/dialog/components/Dialog.web.js

```diff
@@ -0,0 +1,267 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import AKButton from '@atlaskit/button';
+import AKButtonGroup from '@atlaskit/button-group';
+import ModalDialog from '@atlaskit/modal-dialog';
+
+import { hideDialog } from '../actions';
+import { translate } from '../../i18n';
+
+/**
+ * Web dialog that uses atlaskit modal-dialog to display dialogs.
+ */
+class Dialog extends Component {
+
+    /**
+     * Dialog component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * Whether cancel button is disabled. Enabled by default.
+         */
+        cancelDisabled: React.PropTypes.bool,
+
+        /**
+         * Optional i18n key to change the cancel button title.
+         */
+        cancelTitleKey: React.PropTypes.string,
+
+        /**
+         * This is the body of the dialog, the component children.
+         */
+        children: React.PropTypes.node,
+
+        /**
+         * Used to show hide the dialog on cancel.
+         */
+        dispatch: React.PropTypes.func,
+
+        /**
+         * Whether the dialog is modal. This means clicking on the blanket will
+         * leave the dialog open. No cancel button.
+         */
+        isModal: React.PropTypes.bool,
```

Comment:
On mobile all dialogs are modal. Do we have an instance of non modal dialogs on the web?